R/profvis.R

```diff
@@ -163,7 +182,10 @@ profvis <- function(expr = NULL,
     }
     repeat {
       inject(Rprof(prof_output, !!!rprof_args))
-      with_profvis_handlers(expr)
+      cnd <- with_profvis_handlers(expr)
```

Comment:
Not related to this PR but `with_profvis_handlers` captures interrupts so without this there's no way to terminate this loop.